### PR TITLE
Add devcontainer configuration

### LINKED ISSUE #357: Environment by GitHub Codespaces to run teacher's scripts
I customized a Codespace environment for me to run `script/teach-class` script like my own PR below.
https://github.com/lefthandz-githubtraining/training-manual/pull/8

If you would like to use it in this repository too, I will be able to create a pull request here. What do you think?

### PR DESCRIPTION
**Pull request by dzeyelid**
This pull request for #357. Please review and merge them.

- [x] Add dev container configuration
  - [x] Verify running as a Codespace environment
  - [x] Verify running through Remove-Container in Visual Studio Code
- [x] Add how to use dev container environment to READMEs
- [x] Fix text lint errors (`terminology`) that exist from before
  - Change `repo` to `repository`
  - Some parts were responded to by wrapping with `<!-- textlint-disble terminology -->`, `<!-- textlint-enable terminology -->`
    - ref: [Ignoring Text · textlint](https://textlint.github.io/docs/ignore.html)
    - ref: [sapegin/textlint-rule-terminology: Textlint rule to check correct terms spelling](https://github.com/sapegin/textlint-rule-terminology)

resolves #357

### CODE REVIEW

**Comment by amyschoen** [comment]
I don't have a good way to test this currently @ppremk, but I'm not seeing anything here that's going to cause trouble for us.

File: script/README.md [hunk 1]
```diff
@@ -16,10 +16,32 @@ To use these scripts, please follow these steps **first**:
...
+
 - OPTIONAL: If you want to collect feedback from the class, have a URL ready. You can use services such as SurveyMonkey or Google Forms.
 - OPTIONAL: If you're going to offer participants one-on-one appointments, determine the URL participants will use to schedule those. We use [YouCanBook.me](http://youcanbook.me). You'll need the URL later.
 
+### OPTIONAL: Run scripts on a development container or GitHub Codespaces environment
+
+This repository has Visual Studio Code's development container configuration, so you can run these scripts on Remote-Container or GitHub Codespaces. For details, see:
+
+- [Developing inside a Container using Visual Studio Code Remote Development](https://code.visualstudio.com/docs/remote/containers)
+- [GitHub Codespaces Documentation - GitHub Docs](https://docs.github.com/en/codespaces)
```
Comment (parkerbxyz) [inline script/README.md:34]:
```suggestion
- [GitHub Codespaces Documentation - GitHub Docs](https://docs.github.com/codespaces)
```

File: script/README.md [hunk 2]
```diff
@@ -16,10 +16,32 @@ To use these scripts, please follow these steps **first**:
...
+
+The environment is based on Ubuntu 21.04 and includes below.
```
Comment (parkerbxyz) [inline script/README.md:36]:
```suggestion
The environment is based on Ubuntu 21.04 and includes:
```

File: script/README.md [hunk 3]
```diff
@@ -16,10 +16,32 @@ To use these scripts, please follow these steps **first**:
...
+
+- Bash
+- Python 3.10
+- Node.js 16.x
+- `jq`
+- `envsubst`
+- `http` (by HTTPie)
```
Comment (parkerbxyz) [inline script/README.md:43]:
```suggestion
- `http` ([HTTPie](https://github.com/httpie/httpie))
```

File: script/README.md [hunk 4]
```diff
@@ -30,8 +52,14 @@ Here's how to get your local environment up and running:
...
    - the title of your newly created training organization
    - your username, or the username of a shared teaching account
+
+   <!-- textlint-disable terminology -->
+
    - your personal access token (PAT) or the PAT of the shared teaching account that includes the scopes **repo** and **delete_repo**.
    _Note: If your organization has SAML SSO enabled, you would need to [authorize your personal access token for use with SAML SSO](https://docs.github.com/en/github/authenticating-to-github/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on)._
+
+   <!-- textlint-enable terminology -->
+
```
Comment (parkerbxyz) [inline script/README.md:62]: This textlint rule has been [fixed in version 3.0.1](https://github.com/sapegin/textlint-rule-terminology/issues/38#issuecomment-1143205870) and will be included in the next [super-linter](https://github.com/github/super-linter) release. I’ll open a new PR to remove the rule exceptions/workarounds once that release is available.

File: README.md [hunk 5]
```diff
@@ -49,15 +49,19 @@ Docsify doesn't currently support creating PDFs, but you can generate one using
...
 1. Run Calibre once to set up the application.
 
 1. Make sure that the [Calibre command line interface](https://manual.calibre-ebook.com/generated/en/cli-index.html) is available on your path.  This is located at `/Applications/calibre.app/Contents/MacOS/` on macOS if you are running the application from the default location.
 
+<!-- textlint-enable terminology -->
+
 ## Teaching scripts
 
-Scripts that accompany the manual, and their documentation can be found in the [`script/`](script/) directory.
+Scripts that accompany the manual, and their documentation can be found in the [`script/`](script/) directory. This repository includes development container configuration. So you can run the scripts on the Visual Studio developer container or GitHub Codespaces if you have permission to use it.
```
Comment (parkerbxyz) [inline README.md:64]:
```suggestion
Scripts that accompany the manual, and their documentation can be found in the [`script/`](script/) directory. This repository includes development container configuration. So you can run the scripts on the Visual Studio developer container or [GitHub Codespaces](https://github.com/features/codespaces).
```

**Review by parkerbxyz — COMMENTED** [review]
Just a few suggested changes and this will be good to go. Thanks, @dzeyelid! ✨

**Comment by dzeyelid** [comment]
@ppremk @amyschoen @parkerbxyz Thank you for supporting this PR!
  I accepted your accurate suggestions by @parkerbxyz .

**Review by parkerbxyz — APPROVED** [review]

**Pull request merged**